independent-projects/bootstrap/app-model/pom.xml

```diff
@@ -83,4 +83,20 @@
             </exclusions>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jboss.jandex</groupId>
+                <artifactId>jandex-maven-plugin</artifactId>
```

Comment:
It'd be better to add a version. As in
https://github.com/quarkusio/quarkus/blob/main/independent-projects/tools/registry-client/pom.xml#L49
https://github.com/quarkusio/quarkus/blob/main/independent-projects/tools/pom.xml#L88

Reply:
Good catch! I added the version in the parent 👍🏻